larq/conftest.py

```diff
@@ -14,14 +14,20 @@ def eager_mode():
 def graph_mode():
     """pytest fixture for running test in graph mode"""
     with context.graph_mode():
-        yield
+        with tf.compat.v1.Session().as_default():
```

Comment:
What is this for?

Reply:
This is so that [`evaluate`](https://github.com/larq/larq/pull/312/files#diff-915b33c29d82e4e01202fe2566e0f20fR26) can evaluate the tensor in a shared session per unittest.